roles/test_report_send/doc/env2vars.md

```
@@ -0,0 +1,293 @@
+1. an extensive set of environment variables
+2. single environment variable, pointing at a file with structured JSON payload
+3. both
+
+Still, each CI system has its own considerations, terminology and this data is kept in different structures/environment variables.
+
+## Solution overview
+
+We not have full control of our reporting systems, since we are only users.
+So we cannot rely on our capability to post-ingest processing.
+In short, we 
+1. can't admin the reporting (regulation, policies)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)